### PR TITLE
touchdetector: assorted updates

### PR DESCRIPTION
**Pull request by matz-e**
- touchdetector: require caliper 2.8.0+, not master
- touchdetector: fall back to specify tbb very well

### CODE REVIEW

File: bluebrain/repo-bluebrain/packages/touchdetector/package.py [hunk 1]
```diff
@@ -118,3 +118,6 @@ def cmake_args(self):
                 )
 
         return args
+
+    def setup_build_environment(self, env):
+        env.prepend_path("CMAKE_PREFIX_PATH", self.spec["tbb"].package.component_prefix)
```
Comment (1uc) [inline bluebrain/repo-bluebrain/packages/touchdetector/package.py:125]: What is the reason for this line? Naively, I'd expect Spack to automatically ensure packages can be found by CMake.

Comment (matz-e) [reply bluebrain/repo-bluebrain/packages/touchdetector/package.py:125]: Intel likes to hide its software... Installing from source, Spack installations contain a `lib` link that likes to disappear when creating a binary cache. This should be more robust, as it directly references the target of said link.

Comment (1uc) [reply bluebrain/repo-bluebrain/packages/touchdetector/package.py:125]: Please consider documenting the reason.

**Review by 1uc — DISMISSED** [review]

File: bluebrain/repo-bluebrain/packages/touchdetector/package.py (hunk 1 above)
Comment (matz-e) [reply bluebrain/repo-bluebrain/packages/touchdetector/package.py:125]: Sure, done!

**Review by 1uc — APPROVED** [review]

**Pull request merged**